spring-cloud-dataflow-test/pom.xml

```diff
@@ -31,6 +31,16 @@
 			<groupId>org.testcontainers</groupId>
 			<artifactId>junit-jupiter</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-jdbc</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-csv</artifactId>
+			<version>1.10.0</version>
+		</dependency>
```

Comment:
We need to update the scope to `Test` for these dependencies.

Reply:
`CSVLoader` is in `src/main/java` in `spring-cloud-dataflow-test` which is in `test` scope in `spring-cloud-dataflow-server`.

From `./mvnw dependency:tree -pl spring-cloud-dataflow-server`

```
[INFO] +- org.springframework.cloud:spring-cloud-dataflow-test:jar:2.11.3-SNAPSHOT:test
[INFO] |  +- org.springframework.cloud:spring-cloud-dataflow-core-dsl:jar:2.11.3-SNAPSHOT:compile
[INFO] |  +- uk.org.webcompere:system-stubs-jupiter:jar:1.2.0:test
[INFO] |  |  \- uk.org.webcompere:system-stubs-core:jar:1.2.0:test
[INFO] |  +- org.springframework:spring-jdbc:jar:5.3.32:compile
[INFO] |  |  +- org.springframework:spring-beans:jar:5.3.32:compile
[INFO] |  |  \- org.springframework:spring-tx:jar:5.3.32:compile
[INFO] |  \- org.apache.commons:commons-csv:jar:1.10.0:test
```